test/cmd/lfstest-gitserver.go

```diff
@@ -76,6 +76,14 @@ func main() {
 			return
 		}
 
+		if strings.Contains(r.URL.Path, "/info/lfs/locks") {
```

Comment:
I'm not going to worry about this since `master` diverged when you fixed the locking relative path issue.